web/core/components/issues/issue-layouts/list/base-list-root.tsx

```diff
@@ -107,6 +112,21 @@ export const BaseListRoot = observer((props: IBaseListRoot) => {
     [fetchNextIssues]
   );
 
+  //kanbanFilters and EIssueFilterType.KANBAN_FILTERS are used becuase the state is shared between kanban view and list view
+  const handleCollapsedGroups = (toggle: "group_by" | "sub_group_by", value: string) => {
```

Comment:
please wrap this with `useCallback` and add necessary dependencies.

Also for list view, it doesn't need the first variable, since it is always `group_by`

Reply:
Made the necessary changes: 
- Removed sub_group_by property from list view
- Used useCallback for both list view and kanban view's filter handling function
- Made naming consistent, Renamed kanbanFilters -> collapsedGroups and handleKanbanFilters -> handleCollapsedGroups for both list and kanban view